addons/beehave/nodes/beehave_tree.gd

```diff
@@ -1,3 +1,10 @@
 ## Base class for all parts of the behaviour tree.
 class_name BeehaveTree extends Node
 @icon("../icons/icon.svg")
+
```

Comment:
Remove extra line

Reply:
I was trying to follow [godot's style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html#blank-lines) with two spaces after class definitions.

Reply:
> Surround functions and class definitions with two blank lines:

interesting. I might follow that, too.